### DISCUSSION TITLE
control TPIC6B595

### DISCUSSION THREAD

**Discussion by BenI2C** [arendst/Tasmota#13302] [Ideas]
I have at home about 10 devices with Tasmota.
Now I look for the control  for my 2.3"  4 digit 7 segment display with   4x TPIC6B595 serial->parr .  
Is this possible with Tasmota?

**Comment by sfromis** [comment]
Not out of the box, but If you create a driver.....

**Reply by barbudor** [reply]
Tasmota already supports TM1637, TM1638 and MAX2119 7-segment and led matrix controllers, including Adafruit Led backpack
May be not needed to re-invent the wheel and use a supported driver

**Comment by BenI2C** [comment]
@barbudor . Ok  but there is a essential difference between those ICś. Tasmota support the multiplex system only ->  lower brigthnes. With the  TPIC6B595 or also 74HC595 each segment have 100 % current possible. You can also make long led strings with higher brigthnes. 
@sfromis. to create a driver, can and will somebody help me ?

**Reply by barbudor** [reply]
Basic standalone driver can be created in Scripting-Language or Berry (Berry is ESP32 only)
More easy than diving into C/C++

Discord user pkkrusty is looking for something similar. You may team up

**Comment by BenI2C** [comment]
How i can contact  pkkrusty ?

**Reply by barbudor** [reply]
on Discord. 
No idea if he has a Github account and what would it be

**Reply by sfromis** [reply]
Someone with the handle [pkkrusty](https://github.com/pkkrusty) is present on Github, including Tasmota-related activity.....
